core/src/main/java/org/apache/calcite/sql/fun/SqlLibrary.java

```diff
@@ -73,7 +73,9 @@ public enum SqlLibrary {
   POSTGRESQL("p", "postgresql"),
   /** A collection of operators that are in Apache Spark but not in standard
    * SQL. */
-  SPARK("s", "spark");
+  SPARK("s", "spark"),
+  /** A collection of operators that are in Snowflake but not in standard SQL. */
+  SNOWFLAKE("s", "snowflake");
```

Comment:
I could be wrong, but do these single chars point to the symbol we use in `reference.md` to indicate the supported libraries? 
If so should they be distinct symbols for each or its okay to use the same, I see `s` is used for Spark, Snowflake and Spatial but 'MSSQL' = `q` and 'MYSQL' = `m`

Reply:
Just saw `reference.md` defines its own symbols for each library, just wanted to confirm what these chars are intended for.

Reply:
List should be alphabetical.

Reply:
Changed to `f` and re-ordered list.